apps/web/prisma/zod/attendee.ts

```diff
@@ -1,6 +1,6 @@
-import * as z from "zod";
```

Comment:
Prettier should ignore these auto-generated files. Everything in ` apps/web/prisma/zod` basically.